### PR TITLE
Allow matlab to run on pitzer cluster

### PR DESCRIPTION
**Pull request by gbyrketosc**
PR to add functionality for Matlab to be launched on the Pitzer Cluster.

### CODE REVIEW

File: form.yml.erb [hunk 1]
```diff
@@ -15,6 +13,9 @@ form:
   - node_type
   - bc_vnc_resolution
   - bc_email_on_started
+cluster:
+    - "owens"
+    - "pitzer"
```
Comment (johrstrom) [inline form.yml.erb:18]: Can you move this bit to the top. It's a little nit-picky, but it's just to keep uniformity with all the other apps and docs and so on. Lots of folks look at our apps for examples and best practices (such as they are).

File: form.yml.erb [hunk 2]
```diff
@@ -38,22 +38,78 @@ attributes:
...
+      widget: select
+      label: "Node type"
+      help: |
+        - **Standard Compute** <br>
+          These are standard HPC machines. Owens has 648 of these nodes with 40
+          cores and 128 GB of memory. Pitzer has 224 with 40 cores and
+          340 with 48. All pitzer nodes have 192 GB of RAM. Chosing any will decrease
+          your wait time.
+        - **GPU Enabled** <br>
+          These are HPC machines GPUs. Owens has 160 nodes with 1 [NVIDIA Tesla P100 GPU]
```
Comment (johrstrom) [inline form.yml.erb:50]:
```suggestion
          These are HPC machines with GPUs. Owens has 160 nodes with 1 [NVIDIA Tesla P100 GPU]
```

**Comment by johrstrom** [comment]
Not all versions are available on pitzer

If you're on a pitzer login node and run `module spider matlab` you'll see the versions available on that system.

```text
  matlab:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     Versions:
        matlab/r2018a
        matlab/r2018b
        matlab/r2019a
        matlab/r2019b
        matlab/r2020a
```

**Comment by gbyrketosc** [comment]
@johrstrom, @Oglopf 

This is ready for review again.

Thanks,
-gerald

**Comment by johrstrom** [comment]
Thanks! Now we just need a changlog update and we can release the next version.

**Pull request merged**